.asf.yaml

```diff
@@ -40,4 +40,80 @@ github:
     # disable merge button:
     merge:   false
     # disable rebase button:
-    rebase:  false
\ No newline at end of file
+    rebase:  false
+  protected_branches:
+    master:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # Contexts are the names of checks that must pass. These names are
+        # found in the "name:" field of ./github/workflows/*.yaml files
+        # Note: here is the list of intentionally excluded contexts:
```

Comment:
This isn't correct information. See https://github.com/apache/pulsar/pull/14226#issuecomment-1035997475 for the current information. The contexts gets passed as-is in the asfgit python code. I'll check where to get the correct context names.

Reply:
Thank you for finding this. I struggled to find documentation on this piece, and determined it was the name based on the the way the GitHub UI takes input. This makes much more sense.